bobtemplates/odoo/hooks.py

```diff
@@ -151,9 +154,29 @@ def post_render_model(configurator):
 # addon hooks
 #
 
+def _set_oca_website(configurator):
+    website = configurator.variables['copyright.website']
+    website = OCA_WEBSITE
+    try:
+        result = subprocess.check_output(
+            "git config --get remote.origin.url",
+            shell=True,
+            stderr=subprocess.STDOUT,
+        )
+        if result:
+            regex = "[^\/]+(?=\.git$)"
```

Comment:
I'm not convinced by this regex

Reply:
@sbidoul This gets what's between a slash '/' and '.git'. I'm not specialized in regex... but that seems to me sufficent